aiida_common_workflows/cli/launch.py

```diff
@@ -104,15 +109,15 @@ def cmd_relax(  # pylint: disable=too-many-branches
                 'Either provide it with the -X option or make sure such a code is configured in the DB.'
             )
 
-        engines[engine] = {
-            'code': code.full_label,
-            'options': {
-                'resources': {
-                    'num_machines': number_machines[index],
-                },
-                'max_wallclock_seconds': wallclock_seconds[index],
-            }
+        all_options = {
+            'resources': {
+                'num_machines': number_machines[index],
+            },
+            'max_wallclock_seconds': wallclock_seconds[index],
         }
+        all_options.update(engine_options)
```

Comment:
With the current design the specified options will be applied to all engines, which is maybe fine for now. But would it make sense to make it possible to specify options per engine? For example if one engine requires one partition/account but the second engine different ones, in this way you cannot specify separately. Only downside for requiring to map it directly on the engine name, is that for the single engine workchains you always need to specify it even though it is the only one and for those with two, you need to specify both to override even if you want to override with the same settings. Still, I think the more flexible version is maybe better?